source/user-manual/api/rbac/reference.rst

```diff
@@ -123,6 +123,9 @@ This reference also contains a set of default roles and policies that can be imm
     - `Vulnerability`_
         - `vulnerability:read`_
 
+    - `Events`_
+        - `event:ingest`_
+
```

Comment:
These new references could be placed after `Decoders` and before `Group` to maintain the alphabetical order of the list.